README.md

```diff
@@ -2,16 +2,38 @@
 
 Source plugin for pulling documents into Gatsby from a Strapi API.
 
-> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v3 at the moment. We are currently working on a v4 compatible version.
+> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v4. For v3 use this [release](https://www.npmjs.com/package/gatsby-source-strapi/v/1.0.3)
+
+<details>
+<summary><strong>Table of contents</strong></summary>
+
+- [gatsby-source-strapi](#gatsby-source-strapi)
+  - [Install](#installing-the-plugin)
+    - [Basic usage](#basic-usage)
+    - [Advanced usage](#advanced-usage)
+      - [Deep queries populate](#deep-queries-populate)
+      - [Draft content](#draft-content)
+      - [Image optimisation](#image-optimisation)
+      - [Rich text field](#rich-text-field)
+      - [Components](#components)
+      - [Dynamic zones](#dynamic-zones)
```

Comment:
Can you add a localizations section and accompanying documentation?

Reply:
cc/ @remidej @markkaylor

Reply:
Would it be okay to add this in a future release?

Because I think it will be easier once we'll have updated the Gatsby corporate starter which uses i18n

Reply:
That'd be a cool starter to have! I've got alpha 4 working with my test site and it's working well except for a few localization bits. Doing those in a discrete release could be a nice milestone

Reply:
May I chime in here? I'm just a regular developer watching the PR and tries it in development. While a starter is great to reduce startup time and complexity it's IMHO a _nice to have_ while undocumented features makes the lib really hard to use until you release a new starter. I don't want to sound demanding here, you do a great job on this lib, I just wanted to give some user input if you find it helpful. 💚